libs/blocks/locui-create/input-locale/index.js

```diff
@@ -146,11 +151,42 @@ export default function useInputLocale() {
 
   const errorPresent = () => Object.keys(activeLocales).length > 0;
 
-  const handleNext = () => {
+  const prefillActionAndWorkflow = (languages) => {
+    const storedLanguages = project.value?.languages ?? [];
+    if (storedLanguages.length < 1) {
+      return languages.map((lang) => ({ ...lang, action: project.value.type === LOCALIZATION_TYPES.translation ? 'Translate' : 'Rollout', workflow: '' }));
+    }
+    let i = 0;
```

Comment:
Maybe use a better variable name than `i`

Reply:
Thanks. Fixed the comment